<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="MainLayout" rowstretch="0" columnstretch="0">
      <item row="0" column="0">
       <widget class="QSplitter" name="splitter">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="midLineWidth">
         <number>0</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="opaqueResize">
         <bool>true</bool>
        </property>
        <property name="handleWidth">
         <number>4</number>
        </property>
        <property name="childrenCollapsible">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="leftLayout" native="true">
         <property name="sizeIncrement">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <layout class="QGridLayout" name="gridLayout_4" rowstretch="0,0" columnstretch="0,0,0,0">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="1" column="3">
           <layout class="QGridLayout" name="layoutFileTags">
            <property name="spacing">
             <number>0</number>
            </property>
            <item row="0" column="0" colspan="2">
             <widget class="QLabel" name="labelFileTags">
              <property name="text">
               <string>Теги файла</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0" colspan="2">
             <widget class="QLineEdit" name="lineAddTagToFile"/>
            </item>
            <item row="3" column="0">
             <widget class="QPushButton" name="buttonAddTagToFile">
              <property name="text">
               <string>Добавить</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0" colspan="2">
             <widget class="QListWidget" name="listFilesTags">
              <property name="minimumSize">
               <size>
                <width>180</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QPushButton" name="buttonDeleteTagFromFile">
              <property name="text">
               <string>Удалить</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0" colspan="2">
             <layout class="QGridLayout" name="FileListGrid"/>
            </item>
           </layout>
          </item>
          <item row="0" column="3">
           <layout class="QGridLayout" name="layoutTagSearch">
            <property name="spacing">
             <number>0</number>
            </property>
            <item row="2" column="0">
             <widget class="QLineEdit" name="lineAddTagToSearch"/>
            </item>
            <item row="2" column="1">
             <widget class="QPushButton" name="buttonDeleteTagFromSearch">
              <property name="text">
               <string>Удалить</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0" colspan="2">
             <widget class="QListWidget" name="listSearchTag">
              <property name="minimumSize">
               <size>
                <width>180</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="0" column="0" colspan="2">
             <widget class="QLabel" name="labelTagSearch">
              <property name="text">
               <string>Поиск по тегам</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0" rowspan="2">
           <layout class="QGridLayout" name="layoutAllTags">
            <property name="spacing">
             <number>0</number>
            </property>
            <item row="4" column="0">
             <widget class="QCheckBox" name="checkBoxVideo">
              <property name="text">
               <string>Video</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QCheckBox" name="checkBoxImage">
              <property name="text">
               <string>Image</string>
              </property>
             </widget>
            </item>
            <item row="4" column="2">
             <widget class="QCheckBox" name="checkBoxUnknow">
              <property name="text">
               <string>unknow</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0" colspan="3">
             <widget class="QTreeWidget" name="treeAllTags">
              <property name="minimumSize">
               <size>
                <width>180</width>
                <height>0</height>
               </size>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::ExtendedSelection</enum>
              </property>
              <property name="sortingEnabled">
               <bool>true</bool>
              </property>
              <property name="expandsOnDoubleClick">
               <bool>false</bool>
              </property>
              <property name="columnCount">
               <number>1</number>
              </property>
              <attribute name="headerShowSortIndicator" stdset="0">
               <bool>true</bool>
              </attribute>
              <column>
               <property name="text">
                <string notr="true">Тэги</string>
               </property>
               <property name="font">
                <font>
                 <stylestrategy>PreferDefault</stylestrategy>
                </font>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="rightLayout" native="true">
         <layout class="QGridLayout" name="gridLayout_3" rowstretch="0,0,0" columnstretch="0,0,0,0,0">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="4">
           <widget class="QCheckBox" name="checkFileSorting">
            <property name="text">
             <string>Сортировка</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QCheckBox" name="checkTumbnailView">
            <property name="text">
             <string>Предпросмотр</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="5">
           <widget class="QListWidget" name="filesViewer">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="3">
           <widget class="QLineEdit" name="lineSearchFile"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="actionAddFiles"/>
    <addaction name="actionOpenFiles"/>
    <addaction name="actionPath"/>
    <addaction name="actionExit"/>
    <addaction name="separator"/>
    <addaction name="buttonAddTagToDb"/>
    <addaction name="buttonDeleteTagFromDb"/>
   </widget>
   <widget class="QMenu" name="menuProfile">
    <property name="title">
     <string>Профили</string>
    </property>
    <widget class="QMenu" name="menuProfiles">
     <property name="title">
      <string>Профили</string>
     </property>
    </widget>
    <addaction name="menuProfiles"/>
    <addaction name="actionAddDelete"/>
   </widget>
   <widget class="QMenu" name="menuFiles">
    <property name="title">
     <string>Files</string>
    </property>
    <addaction name="button_open"/>
    <addaction name="buttonExport"/>
    <addaction name="separator"/>
    <addaction name="actionViewAll"/>
    <addaction name="actionViewExist"/>
    <addaction name="actionViewNotExist"/>
    <addaction name="separator"/>
    <addaction name="actionViewAllNotExist"/>
    <addaction name="actionAddTypeToFile"/>
    <addaction name="buttonClear"/>
    <addaction name="buttonDeleteFIleFromDb"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuProfile"/>
   <addaction name="menuFiles"/>
  </widget>
  <action name="actionAddFiles">
   <property name="text">
    <string>Добавить файлы</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="actionOpenFiles">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="actionAddDelete">
   <property name="text">
    <string>Добавить/Удалить</string>
   </property>
  </action>
  <action name="actiondefault">
   <property name="text">
    <string>default</string>
   </property>
  </action>
  <action name="actionPath">
   <property name="text">
    <string>Путь</string>
   </property>
  </action>
  <action name="button_open">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="buttonExport">
   <property name="text">
    <string>Экспорт</string>
   </property>
  </action>
  <action name="actionViewExist">
   <property name="text">
    <string>Показать существующие</string>
   </property>
  </action>
  <action name="actionViewAllNotExist">
   <property name="text">
    <string>Показать все несуществующие</string>
   </property>
  </action>
  <action name="buttonClear">
   <property name="text">
    <string>Удалить несуществующие</string>
   </property>
  </action>
  <action name="buttonDeleteFIleFromDb">
   <property name="text">
    <string>Удалить</string>
   </property>
  </action>
  <action name="buttonAddTagToDb">
   <property name="text">
    <string>Добавть тег</string>
   </property>
  </action>
  <action name="buttonDeleteTagFromDb">
   <property name="text">
    <string>Удалить тег</string>
   </property>
  </action>
  <action name="actionAddTypeToFile">
   <property name="text">
    <string>Добавить тип к файлам</string>
   </property>
  </action>
  <action name="actionViewNotExist">
   <property name="text">
    <string>Показать несуществующте</string>
   </property>
  </action>
  <action name="actionViewAll">
   <property name="text">
    <string>Показать все</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
